spyder/plugins/findinfiles/widgets.py

```diff
@@ -991,16 +983,15 @@ def __init__(self, parent,
 
         self.result_browser = ResultsBrowser(self, text_color=text_color)
 
-        hlayout = QHBoxLayout()
-        hlayout.addWidget(self.result_browser)
+        # Create main widget.
+        main_widget = QWidget()
+        main_widget_layout = QVBoxLayout(main_widget)
+        main_widget_layout.setContentsMargins(0, 0, 0, 0)
+        main_widget_layout.addWidget(self.result_browser)
+        main_widget_layout.addWidget(self.status_bar)
 
-        layout = QVBoxLayout()
-        left, _x, right, bottom = layout.getContentsMargins()
-        layout.setContentsMargins(left, 0, right, bottom)
-        layout.addWidget(self.find_options)
-        layout.addLayout(hlayout)
-        layout.addWidget(self.status_bar)
-        self.setLayout(layout)
+        # Create plugin layout.
+        self.setLayout(create_plugin_layout(self.find_options, main_widget))

```

Comment:
Perhaps we should rename `create_plugin_layout` because now it's not only used for plugins?